docs/resources/rf_stack.md

```diff
@@ -0,0 +1,194 @@
+---
+subcategory: "Application Orchestration Service (AOS)"
```

Comment:
should this be RF?

Reply:
As the category for provider document, the name is `Application Orchestration Service (AOS)`.
And for others, `RF`. is used.